pipelines/build/openjdk11_pipeline.groovy

```diff
@@ -90,7 +93,7 @@ def buildConfigurations = [
                 os                   : 'linux',
                 additionalNodeLabels : 'centos6',
                 arch                 : 'x64',
-                test                 : false,
+                test                : ['openjdktest', 'systemtest']
```

Comment:
whitespace

Reply:
I approve the PR, but note that we are 'borrowing' the AIX build machine to test on.  This will run tests on the build machine (which has been tagged with appropriate Jenkins tag to automatically run).  I will need to update the test job itself, as its configured specially to run on weekends (and pull a recent nightly build).